wagtail/tests/dummy_sendfile_backend.py

```diff
@@ -0,0 +1,8 @@
+from django.http import HttpResponse
+
+
+def sendfile(request, filename, **kwargs):
+    '''
```

Comment:
Could you change this to tripple double quotes (https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring)